tests/ftests/cgroup.py

```diff
@@ -163,6 +163,9 @@ def create(config, controller_list, cgname, user_name=None,
         if ignore_systemd:
             cmd.append('-b')
 
+        if create_scope:
+            cmd.append('-c')
```

Comment:
The check for mutually exclusive flags `-b` and `-c` check needs to be done here, both can't be set at the same time.

Reply:
We need to update `create_and_validate()` too.

Reply:
Good find.  I chose to add the mutual exclusion check in libcgroup itself rather than the tests.